contracts/src/core/BatcherPaymentService.sol

```diff
@@ -236,30 +281,38 @@ contract BatcherPaymentService is
         );
 
         address signer = noncedHash.recover(signatureData.signature);
-        require(signer != address(0), "Invalid signature");
 
-        UserInfo storage user_data = UserData[signer];
+        if (signer == address(0)) {
+            revert InvalidSignature();
+        }
+
+        UserInfo storage user_data = userData[signer];
```

Comment:
Same here, should probably be signerData or signerInfo or similar

Reply:
Solved in [0191794](https://github.com/yetanotherco/aligned_layer/pull/843/commits/01917949f261623204b62054612eeace2527c0cf).